tasks/buildsources.js

```diff
@@ -107,6 +107,15 @@ module.exports = function(grunt) {
 				config.rawSource = '\n// '+featureName + '\n' + config.rawSource;
 
 				var featurePath = path.join(destFolder, featureName+'.json');
+                var featureDir = path.dirname(featurePath);
+                mkdir(featureDir, function(err) {
```

Comment:
Hi :)
I actually used the sync version. Check line 17
```
var mkdir = require('mkdirp').sync;
```

Perhaps I should keep a mkdir variable to get the dependency only, and then explicitly call mkdirp.sync(dir, opts).
What do you think ?